test/e2e/common/apparmor.go

```diff
@@ -67,6 +67,18 @@ elif ! grep "%[3]s" /proc/self/attr/current; then
   exit 3
 fi`, appArmorDeniedPath, appArmorAllowedPath, appArmorProfilePrefix+f.Namespace.Name)
 
+	if unconfined {
+		profile = apparmor.ProfileNameUnconfined
+		testCmd = `
+if cat /proc/sysrq-trigger | grep "Permission denied"; then
+  echo "FAILURE: reading /proc/sysrq-trigger should be allowed"
+  exit 1
+elif ! grep unconfined /proc/self/attr/current; then
```

Comment:
This check has undesired side-effect -- it prints the line with `unconfined` word to stdout if it will find it. Consider using `-q` option to suppress grep's output.

I'd also suggest to use `fgrep` instead of `grep` because you're searching for a simple string and not regular expression (hence `grep` should be faster).